sdk/v1/client.go

```diff
@@ -233,6 +235,45 @@ func (cli *Client) GetJob(ctx context.Context, reqheader client.Headers, jobID s
 	return &respHeaders, &job, nil
 }
 
+func (cli *Client) GetJobs(ctx context.Context, reqheader client.Headers, options client.Options) (*client.RespHeaders, *models.Jobs, error) {
+	if reqheader.ServiceAuthToken == "" {
+		reqheader.ServiceAuthToken = cli.serviceToken
+	}
+
+	configValues, _ := config.Get()
+	validOffset, err := cli.ValidateOptions(options.Offset)
+	if err != nil || validOffset == "0" {
+		validOffset = strconv.Itoa(configValues.DefaultOffset)
+	}
+
+	validLimit, err := cli.ValidateOptions(options.Limit)
+	if err != nil || validLimit == "0" {
+		validLimit = strconv.Itoa(configValues.DefaultLimit)
```

Comment:
If someone has specified a limit of 0 then this would cause the endpoint to have no limit and therefore would list all the jobs even if there were more than the maximum that should be returned. So, to stop this from happening we could use the default limit as you have done. However, I think it might be better to use the default maximum limit.
```suggestion
		validLimit = strconv.Itoa(configValues.DefaultMaxLimit)
```